csci-499-sp24/FaceStylizer
# Integrating Google Authentication to Sign Up Form

Testing Instructions:

1. run both client and server with npm run dev
2. click on the sign in button
3. click on sign in with google with your google account
4. you should be redirected to http://localhost:3000/home 
5. you should see Hi, "user name" on the upper right hand corner
